Condensed forms

daliboris opened the pull request:
includes
- example (Old-Czech _leleti (sě)_)
- bibliography
- chapter _Forms_ section
- improved XSLT for books with multiple authors

TODO:
- [x] add one paragraph description of condensed forms
- [x] check visibility of the image

daliboris committed:
feat(docs): example of condensed forms

daliboris committed:
feat(docs): add MSS dictionary source

daliboris committed:
WIP: feat(docs): description of condensed forms

daliboris committed:
feat(html): improve multiple authors generation for monographs

daliboris committed:
WIP: feat(docs): image URL with caption

daliboris committed:
feat(docs): condensed forms documentation

daliboris committed:
fix(doc): move Condensed forms before Multiword expressions

daliboris committed:
fix(doc): add <gram type="valency" value="reflexive" />; simplify sense definition

daliboris committed:
fix(doc): remove redundant <idno>

daliboris committed:
fix(doc): comma add before last name

daliboris committed:
feat(doc): documentation next version

ttasovac committed:
feat: add documentation on condensed forms

ttasovac commented:
thanks @daliboris! I added narrative documentation to this PR. I also fixed the image not showing up —  I had to use a different URL from imgur with .png at the end so that we embedd the image itself rather than a whole page from imgur for the given image.

I think we're still missing one thing though: an entry in our revision history —  `revisionDesc` in `TEILex0.odd` which will say that we've added documentation on condensed forms. Could you please go back to our notes and make sure we've documented properly how we're doing our revision history, bumping up the version number etc.

ttasovac committed:
feat(doc): added type values to lbl

As suggested by @laurentromary in https://github.com/DARIAH-ERIC/lexicalresources/issues/227#issuecomment-2914994910, I added types on lbl.

ttasovac commented:
@daliboris for the sake of consistency, I have added two types on lbs in our example: `condensedLemmas` and `condensedInflectedForms`

ttasovac committed:
Add changelog entry for condensed forms documentation

Updated the revision history to include documentation on encoding condensed forms such as 'leleti (sě)'.

ttasovac committed:
Bump TEILex0 schema version to 0.9.5-dev

Updated the edition number in TEILex0.odd to 0.9.5-dev and revised the comment to remind maintainers to remove the dev suffix before publishing a new release.

ttasovac committed:
Regenerate TEILex0.rng and HTML documentation

ttasovac reviewed (approved):
All good. I have bumped up the version number and documented the changes in our change log. Also, the base branch was `master` but it should be `dev`.
